databases/week3/hyf_meal_sharing_db.sql

```diff
@@ -0,0 +1,155 @@
+where id = 1;
+
+DELETE 
+from reviews
+where id = 1;
+
+
+-- addition of meals
+insert into meals (title, description, location, `when`, max_reservations, price, created_date) values ('tacos', 'corn tortilla with meat', 'Mexico', NOW(), 15, 10, '2020-12-01');
+insert into meals (title, description, location, `when`, max_reservations, price, created_date) values ('fish and chips', 'fried fish and fries', 'England', NOW(), 5, 100, '2021-01-01');
+insert into meals (title, description, location, `when`, max_reservations, price, created_date) values ('curry fries', 'fries with curry and sauce', 'Germany', NOW(), 5, 40, '2021-01-15');
+insert into meals (title, description, location, `when`, max_reservations, price, created_date) values ('arepas', 'stuffed corn tortilla', 'Colombia', NOW(), 10, 30, '2020-11-01');
+insert into meals (title, description, location, `when`, max_reservations, price, created_date) values ('flaeskesteg', 'roast pork', 'Denmark', NOW(), 5, 10, '2021-02-01');
+
+-- addition of reservations
+insert into reservations (number_of_guests, meal_id, created_date, contact_phonenumber, contact_name, contact_email) values (5, 4, '2021-05-07', 123456789, 'Rachel Green', 'rachel@friends.com');
+insert into reservations (number_of_guests, meal_id, created_date, contact_phonenumber, contact_name, contact_email) values (2, 4, '2021-05-08', 987654321, 'Rory Gilmore', 'rory@gg.com');
+insert into reservations (number_of_guests, meal_id, created_date, contact_phonenumber, contact_name, contact_email) values (4, 3, '2021-05-07', 00000000, 'John Dorian', 'jd@scrubs.com');
+insert into reservations (number_of_guests, meal_id, created_date, contact_phonenumber, contact_name, contact_email) values (5, 3,'2021-05-10', 11111111, 'Ted Mosby', 'ted@himym.com');
+insert into reservations (number_of_guests, meal_id, created_date, contact_phonenumber, contact_name, contact_email) values (2, 7, '2021-05-20', 22222222, 'Frodo Baggins', 'frodo@lotr.com');
+
+-- addition of reviews
+insert into reviews (title, description, meal_id, stars, created_date) values ('Great experience', 'The host was really nice', 4, 4, current_date());
+insert into reviews (title, description, meal_id, stars, created_date) values ('Can improve', 'We had to wait a long time', 5, 2, '2021-05-02');
+insert into reviews (title, description, meal_id, stars, created_date) values ('Not pet friendly', 'My pet Smeagol had to wait outside', 6, 1, '2021-02-02');
+insert into reviews (title, description, meal_id, stars, created_date) values ('Dirty forks', 'My friend Monica had a problem with some dirty dishes', 7, 3, '2021-12-02');
+insert into reviews (title, description, meal_id, stars, created_date) values ('Good coffee', 'Coffee was great', 5, 4, '2021-07-02');
+
+-- queries
+select *
+from meals
+where price < 50;
+
+select meals.id, meals.title, meals.max_reservations, SUM(reservations.number_of_guests), (meals.max_reservations - reservations.number_of_guests) AS available_space
```

Comment:
I see a lot of nulls when i run this query:

![image](https://user-images.githubusercontent.com/609864/116156353-a757b300-a6eb-11eb-9777-1361ad1b826a.png)

Check this COALESCE() function: https://dev.mysql.com/doc/refman/8.0/en/comparison-operators.html#function_coalesce ;)